ocaml/xapi/console.ml

```diff
@@ -34,6 +34,51 @@ type address =
 
 (* console is listening on a Unix domain socket *)
 
+(* This module enforces connection limits for VM consoles.
```

Comment:
The wording is a bit more general than the implementation. It sounds like various limits are possible but in fact it it either unlimited or at most one per VM. An alternative implementation could have maintained a count of active connections per VM and enforce an upper limit. Such an implementation would have used a hash table.